spec/models/validations/sales/household_validations_spec.rb

```diff
@@ -4,8 +4,8 @@
   subject(:household_validator) { validator_class.new }
 
   let(:validator_class) { Class.new { include Validations::Sales::HouseholdValidations } }
-  let(:record) { build(:sales_log, saledate: log_date) }
-  let(:log_date) { Time.zone.local(2023, 4, 1) }
+  let(:record) { build(:sales_log, saledate:) }
+  let(:saledate) { Time.zone.local(2023, 4, 1) }
```

Comment:
Is this able to be now?

Reply:
This fails one validation, I've set it to now and moved the date into that validation test context